### PR TITLE
Use new GitHub Actions CI

### PR DESCRIPTION
**Pull request by TejasQ**
This PR:

- uses the new GitHub actions CI
- removes the useless `dev-server` folder
- removes Netlify lambdas (we have not been using these for a month already)
  - 💡 separate task idea: let's use GitHub Actions later for Slack notifications

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - name: Test
+        run: yarn test:unit
+      - name: Use npm client
+        if: github.head_ref == 'master'
+        uses: actions/npm@master
+      - name: Publish to npm@next
+        if: github.head_ref == 'master'
+        run: |
+          npm version $(npm show . version)-$(git rev-parse --short HEAD) --no-git-tag-version --unsafe-perm
+          npm publish --tag next
```
Comment (stereobooster) [inline .github/workflows/ci.yml]: `yarn publish`?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          yarn run build:package
+      - name: Test
+        run: yarn test:unit
+      - name: Use npm client
+        if: github.head_ref == 'master'
+        uses: actions/npm@master
+      - name: Publish to npm@next
+        if: github.head_ref == 'master'
+        run: |
+          npm version $(npm show . version)-$(git rev-parse --short HEAD) --no-git-tag-version --unsafe-perm
```
Comment (stereobooster) [inline .github/workflows/ci.yml]: does this work in detached head `git rev-parse --short HEAD`?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (TejasQ) [reply .github/workflows/ci.yml]:
```suggestion
          yarn publish --tag next
```

File: .github/workflows/ci.yml (hunk 2 above)
Comment (TejasQ) [reply .github/workflows/ci.yml]: Yes.

Comment (TejasQ) [reply .github/workflows/ci.yml]: image

**Pull request merged**